sidecar.sh

```diff
@@ -106,6 +152,11 @@ function get_access_token() {
     export ACCESS_TOKEN="$ACCESS_TOKEN"
 }
 
+function get_manifest_service_access_token() {
```

Comment:
why do we need this instead of just `get_gen3_access_token()`

Reply:
Leftover code from before we met and I realized that I didn't need a different access token since the metadata also resides in manifestservice, I removed it and refactored the name of the function to its original name. Thank you for catching this.

Reply:
I think we can remove this function entriely

Reply:
Removed, thanks